keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -10,25 +12,38 @@ class RetinaNetImageConverter(ImageConverter):
 
     def __init__(
         self,
-        image_size=None,
-        scale=None,
-        offset=None,
+        bounding_box_format,
```

Comment:
should we have a default for this?

Reply:
I mean we can have same as default format used for weights, but it can vary as per user dataset.

Reply:
Yeah, but I think a default is still probably helpful? I see a lot of `bounding_box_format="xyxy",` defaults in Keras.

Reply:
Our default weights are in `yxyx`. But yes we can have `xyxy`(torch) follows or `yxyx`(tensorflow) follows this. All the tensorflow datasets will have `yxyx`, I would say `yxyx` would be good option.

Reply:
Done